api/EventTarget.json

```diff
@@ -255,10 +255,10 @@
                   "version_added": "≤18"
                 },
                 "firefox": {
-                  "version_added": true
+                  "version_added": "49"
```

Comment:
Confirmed by testing http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=9840 in Firefox 48 and 49. One has to test `{ capture: false}` as well since if the options dictionary isn't supported at all, any object just gets interpreted as a boolean.

This also confirms that the `options_parameter` data above is correct.